presto-hive-common/src/main/java/com/facebook/presto/hive/zorder/ZAddressRange.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.hive.zorder;
+
+import java.util.Objects;
+
+/**
+ * The ZAddressRange class contains a minimum and maximum address. Both are inclusive.
+ */
+public class ZAddressRange
+        implements Comparable<ZAddressRange>
+{
+    private final long minAddress;
```

Comment:
We can see minimumValue and maximumValue in other files. The usage of "min" and "minimum", "max" and "maximum" should be consistent.